src/main/scala/analysis/SSA.scala

```diff
@@ -1,426 +1,426 @@
-package analysis
+// TODO: This doesn't seem to be used currently, should it be kept?
```

Comment:
Good question, is this fine to completely remove?

Reply:
Functionality defined is deprecated. File has been removed. [Upcoming merge from yousif-memory-region-analysis]